include/core/boundary_conditions.h

```diff
@@ -678,6 +681,162 @@ namespace BoundaryConditions
     prm.leave_subsection();
   }
 
+  /**
+   * @brief This class manages the boundary conditions for VOF solver
+   * It introduces the boundary functions and declares the boundary conditions
+   * coherently. (TODO update)
+   * The members ... contain double used for bc calculation :
+   *  - if bc type is "..." (... condition), "..." is the
+   * double passed to the deal.ii ConstantFunction
+   */
+
+  template <int dim>
```

Comment:
I don't understand how this is related to this PR?

Reply:
Hum strange, I worked on separated folders for both. Maybe I got mixed up, I will check that right away!